src/raft_log.rs

```diff
@@ -416,7 +416,7 @@ impl<T: Storage> RaftLog<T> {
         }
 
         let length = self.last_index() + 1 - first_index;
-        if low < first_index || high > first_index + length {
+        if high > first_index + length {
```

Comment:
Why remove the first check?

Reply:
I think it has already been checked in here https://github.com/tikv/raft-rs/blob/7123d06fb557d26d5e453e32328ab4f0955a87cd/src/raft_log.rs#L414-L420